app/models/ems_refresh/save_inventory_cinder.rb

```diff
@@ -4,15 +4,7 @@
 #   - cloud_volumes
 #   - cloud_volume_backups
 #   - cloud_volume_snapshots
-#   - vms
-#     - storages (link)
-#     - security_groups (link)
-#     - operating_system
-#     - hardware
-#       - disks
-#       - guest_devices
-#     - custom_attributes
-#     - snapshots
+#   - backing_links
 #
 
 module EmsRefresh::SaveInventoryCinder
```

Comment:
Even so, this comment is on a line not part of this PR, meaning it was already merged elsewhere.  So, I won't let it hold up *this* PR, but it needs to be adjusted in a future PR.

cc @blomquisg